administrator/components/com_config/src/Model/ApplicationModel.php

```diff
@@ -299,6 +299,20 @@ public function save($data)
             'prefix'   => $data['dbprefix'],
         ];
 
+        if (in_array($options['driver'], ['pgsql', 'postgresql']) && !preg_match('#^[a-zA-Z_][0-9a-zA-Z_$]*$#', $options['database']))
+        {
```

Comment:
```suggestion
        if (in_array($options['driver'], ['pgsql', 'postgresql']) && !preg_match('#^[a-zA-Z_][0-9a-zA-Z_$]*$#', $options['database'])) {
```
We have new code style rules since a while (PSR-12).

Reply:
and closing with a comma :wink:

`, $options['database'],)) {`

Reply:
@sandewt No that's wrong. It would be right for array elements, but here it is a function call, so a comma after the last element would cause a syntax error with PHP 7.2, a version which we still have to support with J4.

Reply:
P.S.: And for array elements it is only needed when spreading over multiple lines, so not for the array argument `['pgsql', 'postgresql']` in the `in_array` call.

Reply:
Agree, It's not my day. I'm going for a walk.